src/gpu-compute/gpu_command_processor.cc

```diff
@@ -473,18 +473,27 @@ GPUCommandProcessor::driver()
  */
 
 /**
- * TODO: For now we simply tell the HSAPP to finish the packet,
- *       however a future patch will update this method to provide
- *       the proper handling of any required vendor-specific packets.
- *       In the version of ROCm that is currently supported (1.6)
- *       the runtime will send packets that direct the CP to
- *       invalidate the GPUs caches. We do this automatically on
- *       each kernel launch in the CU, so this is safe for now.
+ * TODO: For now we simply tell the HSAPP to finish the packet and write a
+ * completion signal, if any. However, in the future proper handing may be
+ * required for vendor specific packets.
+ *
+ * In the version of ROCm that is currently supported the runtime will send
```

Comment:
Be explicit about version?

Reply:
FS mode will work on anything 4.0-5.4 right now.  I kind of don't want to have to keep updating a comment. The one I replaced was still referencing ROCm 1.6.  IMO user can check documentation for what "supported" versions means

Reply:
Maybe we should just add a pointer to the documentation here then?

Reply:
OK.  Doesn't seem to be any documentation though, so not sure what to point to right now

Reply:
I think the update is fine then, thanks.